lib/elastomer/client/delete_by_query.rb

```diff
@@ -0,0 +1,82 @@
+module Elastomer
+  class Client
+
+    class DeleteByQuery
+
+      def initialize( client, query, params = nil )
```

Comment:
The style guide specifies no spaces between parentheses in method definitions, the rest of this library notwithstanding (cc @TwP the scofflaw)